spec/models/location_spec.rb

```diff
@@ -26,114 +26,111 @@
   end
 
   describe "#update_location" do
-    let(:district) { FactoryGirl.create(:district) }
-    let(:person)   { FactoryGirl.create(:person, district: district,
-                                                 state:    district.state,
-                                                 zip_code: '03431') }
+    let(:location) { FactoryGirl.create(:location, zip_code: '03431') }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.